core/bus/camel/src/main/java/org/switchyard/bus/camel/CamelExchange.java

```diff
@@ -171,9 +174,29 @@ public void send(Message message) {
             getContext().setProperty(Exchange.MESSAGE_ID, camelMsg.getMessageId());
             initOutContentType();
         }
-
+        copyProperties(camelMsg);
```

Comment:
This shouldn't be done. The Exchange#send() is invoked for every exchange, i.e. the interactions of all the implementations and bindings. as implementation.camel is propagating context properties by itself, this definitely is breaking things.